pom.xml

```diff
@@ -20,24 +20,29 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
+            <version>2.7.5</version>
```

Comment:
There is no reason to update version for separate spring libraries. You can update all spring-libraries in 
```
<parent>
        <groupId>org.springframework.boot</groupId>
        <artifactId>spring-boot-starter-parent</artifactId>
        <version>2.7.4</version>
    </parent>
```
version will be inherited from parent pom file